### DISCUSSION TITLE
Has anyone successfully got Flowise to work on an ARM Mac?

### DISCUSSION THREAD

**Discussion by lancerkind** [FlowiseAI/Flowise#2895] [Q&A]
I folks. I'm trying to get flowise to work on an M2 mac using Docker. I'm seeing some challenges and wondering if anyone has overcome them?

**Comment by lancerkind** [comment]
Yes. Using the Flowise build on Jul 27, 2024, (git tag: flowise-components@1.8.6-24-ge39fd130) 
I was able to install Flowise on an M2 mac using the docker compose instructions.  All of this is running **natively on ARM** without emulation.
<img width="1291" alt="Screenshot 2024-07-27 at 16 56 34" src="https://github.com/user-attachments/assets/3c4dcdd2-e789-4742-bc5a-1308d4e92018">

**Reply by sharif3** [reply]
Bit of a noob, can you expand on how this worls ? Thanks!

**Reply by erlebach** [reply]
And also could you please explain how to use Ollama from Docker? 
@sharif3: to use Docker, you need to install the Docker App on the mac. Then you need the .env file, and docker-compose.yml file (from the FLowise github repo).
